src/components/ColorPicker/tests/ColorPicker.test.tsx

```diff
@@ -13,26 +12,31 @@ const red = {
 };
 
 enum SlidableType {
-  BrightnessSaturation,
+  Color,
   Hue,
+  Alpha,
 }
 
 describe('<ColorPicker />', () => {
-  describe('Saturation/ Brightness pane', () => {
+  describe('Color slider (big square/rectangle)', () => {
     describe('onChange', () => {
-      it('is called when the user mouse downs', () => {
+      it("is called when Slidable's onChange is triggered", () => {
         const spy = jest.fn();
-        mountWithAppProvider(<ColorPicker color={red} onChange={spy} />)
-          .find(Slidable)
-          .at(SlidableType.BrightnessSaturation)
-          .simulate('mousedown');
+        const colorPicker = mountWithApp(
+          <ColorPicker color={red} onChange={spy} />,
+        );
+
+        colorPicker.findAll(Slidable)[SlidableType.Color].trigger('onChange', {
```

Comment:
The enum feels really like a really weird use here. What do you think to removing it and using plain old numbers for indexes?

```suggestion
        colorPicker.findAll(Slidable)[0].trigger('onChange', {
```